internal/storage/storage_handle.go

```diff
@@ -105,6 +108,10 @@ func createHTTPClientHandle(ctx context.Context, clientConfig *storageutil.Stora
 		return nil, fmt.Errorf("client-protocol requested is not HTTP1 or HTTP2: %s", clientConfig.ClientProtocol)
 	}
 
+	if clientConfig.SkipAuth {
```

Comment:
We add tokenSrc while creating http client. Should we also stop the creation of token source while creating httpClient?